<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Application</class>
 <widget class="QMainWindow" name="Application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home library</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuBooks">
    <property name="title">
     <string>Books</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionMakeCopy"/>
    <addaction name="actionEdit"/>
    <addaction name="actionRemove"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionDictionaries"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBooks"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="toolBarActionNew"/>
   <addaction name="toolBarActionOpen"/>
   <addaction name="toolBarActionSave"/>
   <addaction name="toolBarActionSaveAs"/>
   <addaction name="toolBarActionClose"/>
   <addaction name="separator"/>
   <addaction name="toolBarActionAdd"/>
   <addaction name="toolBarActionMakeCopy"/>
   <addaction name="toolBarActionEdit"/>
   <addaction name="toolBarActionRemove"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionMakeCopy">
   <property name="text">
    <string>Make copy</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionDictionaries">
   <property name="text">
    <string>Dictionaries</string>
   </property>
  </action>
  <action name="toolBarActionOpen">
   <property name="text">
    <string>uf07b</string>
   </property>
  </action>
  <action name="toolBarActionSave">
   <property name="text">
    <string>uf0c7</string>
   </property>
  </action>
  <action name="toolBarActionSaveAs">
   <property name="text">
    <string>uf0c7</string>
   </property>
  </action>
  <action name="toolBarActionAdd">
   <property name="text">
    <string>uf0fe</string>
   </property>
  </action>
  <action name="toolBarActionMakeCopy">
   <property name="text">
    <string>uf24d</string>
   </property>
  </action>
  <action name="toolBarActionEdit">
   <property name="text">
    <string>uf044</string>
   </property>
  </action>
  <action name="toolBarActionRemove">
   <property name="text">
    <string>uf146</string>
   </property>
  </action>
  <action name="toolBarActionClose">
   <property name="text">
    <string>uf518</string>
   </property>
  </action>
  <action name="toolBarActionNew">
   <property name="text">
    <string>fff</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
